### ISSUE TITLE
Add Level Zero UR Reference Document

### ISSUE THREAD

**Issue by jandres742** [oneapi-src/unified-runtime#947]
Add reference document in the spec, as we have for CUDA and HIP.

**Comment by jandres742** [comment]
@kbenzie : I see we have reference documents for CUDA and HIP.

Do we want then one for L0 as well (see here https://github.com/oneapi-src/unified-runtime/pull/948).

If so, what is the expected content of these documents, could we add there documentation about environment variables and specific behavior for that adapter?

**Comment by kbenzie** [comment]
> If so, what is the expected content of these documents, could we add there documentation about environment variables and specific behavior for that adapter?

Yes, that's an idea use for these documents. Essentially, any behaviour or requirement which is speicifc to a single adapter should be included. So for example, L0 has it's own SPIR-V environment spec so this document should mention that and link to where the detailed docs can be found.
